code/__HELPERS/names.dm

```diff
@@ -15,7 +15,7 @@ GLOBAL_VAR(church_name)
 
 	return name
 
-// TODO: Remove this. Its always gonna be NAS Trurl
+// AA TODO: Remove this. Its always gonna be NAS Trurl
```

Comment:
Theres a lot of these in the diff

After this PR I plan to just search for all `// AA TODO:` and nail them all in one PR